DigitalLearningSolutions.Web/Helpers/DisplayStringHelper.cs

```diff
@@ -58,7 +58,7 @@ public static string GetNameWithEmailForDisplay(string name, string? email)
         public static string? GetPotentiallyInactiveAdminName(string? firstName, string? lastName, bool? active)
         {
             return !string.IsNullOrEmpty(lastName)
-                ? GetNonSortableFullNameForDisplayOnly(firstName, lastName) + (active == true ? "" : " (inactive)")
+                 ? GetNonSortableFullNameForDisplayOnly(firstName, lastName) + (active == true ? "" : " (inactive)")

```

Comment:
I don't think we need this extra space here. The indentation looks pretty good as it is.

Reply:
The space is need because the response is expected format

Reply:
it is, there is a unit test on the method that expect a result in that format if you remove the space the test will failed and the project will not build this is what I tested if you want me to show you I can